src/Catel.MVVM/Catel.MVVM.Shared/Windows/Helpers/WrapControlHelper.cs

```diff
@@ -225,7 +225,29 @@ public static Grid Wrap(FrameworkElement frameworkElement, WrapOptions wrapOptio
                         button.Command = dataWindowButton.Command;
                     }
 
-                    button.Content = dataWindowButton.Text;
+                    if (dataWindowButton.ContentBindingPath != null)
```

Comment:
Maybe it's easier to create a "BindingDescription" or something like that? For now it will container Path and Converter, but maybe we haven't thought of something, then it's easier to add more functionality in the future.